pynldas2/pynldas2.py

```diff
@@ -84,6 +85,43 @@ def wrapper_decorator(*args, **kwargs):  # type: ignore
         "units": "kg/kg",
     },
 }
+
+NLDAS_VARS_V2 = {
```

Comment:
For this, please follow the same pattern as the NLDAS_VARS, that is, use lower case names that are easy to remember. We don't want users to check the document every time they want to add a variable. The `nldas_name` key is supposed to take care of mapping from user input to NLDAS API. Also, instead of using `002` and `V2`, it's best to use some meaningful names. Since, the only difference between the two is the data source, we can use `NLDAS_VARS_GRIB` and `NLDAS_VARS_NC`, so it's easier to understand which is which.